indium-nodejs.el

```diff
@@ -113,6 +113,27 @@ If no process has been started, or if it was not started using
                 (read-from-minibuffer "Path: ")))
   (indium-nodejs--connect host port path))
 
+(defun indium-remove-leading-slash-from-uri (uri)
```

Comment:
It looks like this function should be private (I would then also move it down).

Reply:
Also, except for commands, all functions are prefixed with `indium-nodejs-`.

Reply:
Probably this can be inlined when changed to use `seq-drop`.